packages/admin-next/dashboard/src/components/inputs/chip-input/chip-input.tsx

```diff
@@ -0,0 +1,144 @@
+import { XMarkMini } from "@medusajs/icons"
+import { Badge, clx } from "@medusajs/ui"
+import {
+  KeyboardEvent,
+  forwardRef,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from "react"
+
+type ChipInputProps = {
+  value?: string[]
+  onChange?: (value: string[]) => void
+  onBlur?: () => void
+  name?: string
+  disabled?: boolean
+  allowDuplicates?: boolean
+  showRemove?: boolean
+  variant?: "base" | "contrast"
+  className?: string
+}
+
+export const ChipInput = forwardRef<HTMLInputElement, ChipInputProps>(
```

Comment:
nit: can we create a last chip on blur (or at least remove the input that hasn't been converted to a chip)?